pkcs11js/pkcs11js.d.ts

```diff
@@ -1173,11 +1173,11 @@ declare module "pkcs11js" {
     const CKM_RSA_PKCS_TPM_1_1: number;
     const CKM_RSA_PKCS_OAEP_TPM_1_1: number;
     const CKM_VENDOR_DEFINED: number;
-
+    
```

Comment:
@andy-ms I fixed it. I've got one question about TSD. New version of TS is available v2.0.3. When is TSD going to use lib.d.ts from the newest version of TS. Cause I've got TS error with old my projects which used v1.8